tests/data/dummy_aea/protocols/fipa/message.py

```diff
@@ -70,39 +70,68 @@ def __init__(self, dialogue_reference: Tuple[str, str] = None,
+        assert self.is_set("message_id")
+        return cast(int, self.get("message_id"))
+
+    @property
+    def target(self) -> int:
+        """Get the target of the message."""
+        assert self.is_set("target")
+        return cast(int, self.get("target"))
+
+    @property
+    def performative(self) -> Performative:  # noqa: F821
+        """Get the performative of the message."""
+        return FIPAMessage.Performative(self.get("performative"))
+
+    @property
+    def query(self):
+        """Get the query of the message."""
+        assert self.is_set("query")
+        return self.get("query")
+
+    @property
+    def proposal(self) -> List:
+        """Get the proposal list from the message."""
+        assert self.is_set("proposal")
+        return cast(List, self.get("proposal"))
+
+    @property
+    def info(self) -> Dict:
+        """Get hte info from the message."""
+        assert self.is_set("info")
+        return cast(Dict, self.get("info"))
+
     def check_consistency(self) -> bool:
         """Check that the data is consistent."""
         try:
-            assert self.is_set("dialogue_reference")
-            dialogue_reference = self.get("dialogue_reference")
-            assert type(dialogue_reference) == tuple
-            dialogue_reference = cast(Tuple, dialogue_reference)
-            assert type(dialogue_reference[0]) == str and type(dialogue_reference[0]) == str
-            assert self.is_set("message_id")
-            assert type(self.get("message_id")) == int
-            assert self.is_set("target")
-            assert type(self.get("target")) == int
-            performative = FIPAMessage.Performative(self.get("performative"))
-            if performative == FIPAMessage.Performative.CFP:
-                assert self.is_set("query")
-                query = self.get("query")
-                assert isinstance(query, Query) or isinstance(query, bytes) or query is None
+            assert type(self.dialogue_reference) == Tuple[str, str]
+            assert type(self.dialogue_reference[0]) == str and type(self.dialogue_reference[1]) == str
+            assert isinstance(self.message_id, int)
+            assert isinstance(self.target, int)
+            if self.performative == FIPAMessage.Performative.CFP:
+                assert isinstance(self.query, Query) or isinstance(self.query, bytes) or self.query is None
                 assert len(self.body) == 5
-            elif performative == FIPAMessage.Performative.PROPOSE:
-                assert self.is_set("proposal")
-                proposal = self.get("proposal")
-                assert type(proposal) == list and all(isinstance(d, Description) or type(d) == bytes for d in proposal)  # type: ignore
+            elif self.performative == FIPAMessage.Performative.PROPOSE:
+                assert type(self.proposal) == list and all(isinstance(d, Description) or type(d) == bytes for d in self.proposal)  # type: ignore
```

Comment:
Hmm worth checking, but I think d can only be of type Description per definitions above..

Reply:
don't think type ignore is needed anymore